.gitignore

```diff
@@ -24,3 +24,4 @@ _site
 string
 .sauce.tid
 .npm
+local/
```

Comment:
We used it as a reference for a background file for the demo. I don't think we need it at this point.